test/config-map.json

```diff
@@ -193,5 +193,18 @@
     "image-types": [
       "ec2-sap"
     ]
+  },
+  "./configs/ostree-filesystem-customizations.json": {
+    "image-types": [
+      "edge-raw-image",
+      "edge-ami",
+      "edge-vsphere",
+      "simplified-installer"
+    ],
+    "distros": [
+      "rhel-92",
+      "rhel-93",
+      "rhel-94"
+    ]
```

Comment:
I'm fine with limiting these but I'd like to investigate what's wrong with 9.0 and 9.1.  If it's some weirdness that is distro/package version specific, it's probably fine, but if it's something we're doing, it might be a symptom of a bigger issue, so it would be good to know.